### DISCUSSION TITLE
Access only while on VPN

### DISCUSSION THREAD

**Discussion by MattTheEmp** [NginxProxyManager/nginx-proxy-manager#1728] [Q&A]
Hello I have Set up my Nginx with several apps. I am able to access normally from open web. I have added VPN subnet to to an access list. when testing via VPN results in a 403.

any thoughts

**Comment by chaptergy** [comment]
Have a look at https://github.com/jc21/nginx-proxy-manager/discussions/1724, maybe that helps.
